srcmedia/scss/cards/_result.scss

```diff
@@ -21,29 +38,36 @@ ol.card-results {
 
     border-top: 0;
     width: $card-width;
-    /* space between cards: subtract three cards and divide remainder by 2 */;
-    margin-right: calc((100% - (#{$card-width} * 3)) / 2);
     margin-bottom: 2.8rem;
 
-    &:nth-child(3n+0) {
-        margin-right: 0;
+    // on tablet and desktop, three cards per line
+    @media (min-width: $breakpoint-s + 1) {
```

Comment:
this is a cool solution...never thought of using nth-child to do the justification this way, but it makes sense!